test/xpath/test_base.rb

```diff
@@ -651,7 +651,7 @@ def test_comparisons
       source = "<a><b id='1'/><b id='2'/><b id='3'/></a>"
       doc = REXML::Document.new(source)
 
-      # NOTE TO SER: check that number() is required
+      # NOTE TO SEE: check that number() is required
```

Comment:
Is that a "note to self" spelled different?

Reply:
Maybe. I guess `NOTE TO SEE` because `E` is placed near by `R` on the qwerty keyboard.

Reply:
SER is the original auther's name: Sean Russell -> SER
See also: https://github.com/ruby/rexml/blob/1e31ffc7c9170255c2a62773ac1e1d90c4991a9d/lib/rexml/rexml.rb#L32

Reply:
@kou My apologies, today I learned.

Reply:
No problem. It's just FYI. :-)